src/lib.rs

```diff
@@ -167,3 +169,16 @@ macro with_args($cmd: ident, $args: ident) {{
         }
     }
 }}
+
+
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    //Just perform some basic integration tests against a server
+    #[test]
+    fn it_works() {
+        assert_eq!(2 + 2, 4);
```

Comment:
Did you mean to add a proper test here?

Reply:
Ah yes, that was my plan. I was about start just adding basic connect/write/read tests, to get rid of running them manually on the side. Basically small applications running against a local server.

I must have commited that one by mistake, sorry.